### ISSUE TITLE
Extra or missing field in Furnishing type

### ISSUE THREAD

**Issue by Victorsitou** [dvaJi/genshin-data#14]
`Furnishing.originalId` is only available in one Furnishing object, but I'm not sure if it's an extra field, or it's missing in the other objects.

https://github.com/dvaJi/genshin-data/blob/088764d161631a5892f7de00c7c1aec9df25c680/src/types/furnishing.ts#L14-L17

**Comment by dvaJi** [comment]
You're right, `Furnishing.originalId` doesn't exist. We should remove it.
